src/forms/output/BenefitText.js

```diff
@@ -0,0 +1,409 @@
+  `$${round$(future.benefitsTotal)} a month. ` +
+  `This is how your benefits might change:`;
+
+  // List of benefits html list items
+  var benefitList = [];
+  var numBenefits = current.benefits.length;
+  for (let benefiti = 0; benefiti < numBenefits; benefiti++) {
+
+    let cBenefit = current.benefits[ benefiti ],
+        fBenefit = future.benefits[ benefiti ];
+
+    benefitList.push(
+      <li key = { cBenefit.label }>
+        {
+          `${cBenefit.label} might change from about ` +
+          `$${round$(cBenefit.amount)} to about ` +
+          `$${round$(fBenefit.amount)} a month.`
+        }
+      </li>
+    );
+  }  // ends for each benefit
+
+  // Feedback button
+  var disclaimer = ([
+    <span key = { `pre-ask` }>
+      { snippets.i_feedbackAsk }
+    </span>,
+    <Button
+      compact
+      key     = { `ask` }
+      size    = { `small` }
+      onClick = { openFeedback }>
+      { snippets.i_submitFeedback }
+    </Button>,
+  ]);
+
+  // Describe how totals change
+  var posDiff    = round$(Math.abs(diff)),
+      lessOrMore = ``;
+  if (diff > 0) {
+    lessOrMore = `That's $${posDiff} more than before.`;
+  } else if (diff < 0) {
+    lessOrMore = `That's $${posDiff} less than before.`;
+  } else if (diff === 0) {
+    lessOrMore = `That's the same as before`;
+  }
+
+  var summaryFuture = 
+    `If this tool is right, you might bring in about ` +
+    `$${round$(future.total)} a month. ${lessOrMore}`;
+
+  // If there was a cliff, how much more will they have
+  // to earn before they'll get more than they are now?
+  var endOfCliffText = null;
+  if (gain.total !== undefined) {
+    endOfCliffText =
+      `The tool says that if you can get to where your household makes about ` +
+      `$${round$(gain.earned)} a month, you could bring in about ` +
+      `$${round$(gain.total - current.total)} more each month all together.`;
+  }
+
+  return (
+    <div>
+
+      {/* For styling, make sure `<p>` isn't last child */}
+      <div>
+        <Header>What could happen?</Header>
+        <p>{ detailsNow }</p>
+        <p>{ detailsFuture }</p>
+        <ul>
+          { benefitList }
+        </ul>
+        <p>{ disclaimer }</p>
+        <span />
+      </div>
+
+      <div>
+        <Header>What could it add up to?</Header>
+        <p>{ summaryFuture }</p>
+        <span />
+      </div>
+      
+      {endOfCliffText === null ? (
+        `After this, the tool says you could keep bringing in more with each raise.`
+      ) : (
+        <div>
+          <Header key = { `gain-header` }>When could things get better?</Header>
+          <p key = { `gain-summary` }>{ endOfCliffText }</p>
+          <span />
+        </div>
+      )}
+
+      {diff <= 0 ? (
+        <div>
+          <hr />
+          <p>If you're worried about this, please search for "social services" in your area to try to find a local case manager.</p>
+        </div>
+      ) : (
+        null 
+      )}
+    </div>
+  );
```

Comment:
Why don't we construct this div and the one above before the return statement, and use an if statement to assign either the divs or the alternate thing (in this case `null`) to a variable, depending on whether there's a cliff? Then we wouldn't need these ternaries.